ui/src/app.css

```diff
@@ -1,3 +1,35 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+html {
+	@apply antialiased;
+}
+
+html.dark {
+	@apply bg-neutral-800 text-white;
+}
+
+@media screen and (max-width: 1024px) {
+    .grid-cols-fieldmonitor {
+        grid-template-columns: minmax(auto, 1fr) 1fr 1fr 1fr auto minmax(auto, 1fr);
+    }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+    .grid-cols-fieldmonitor {
+        grid-template-columns: minmax(auto, 1fr) 1fr 1fr 1fr auto minmax(auto, 1fr);
+        width: fit-content;
+    }
+}
+
+@media screen and (min-width: 1024px) {
+    .grid-cols-fieldmonitor {
+        grid-template-columns: minmax(auto, 1fr) 1fr 1fr 1fr auto minmax(auto, 1fr) minmax(auto, 1fr) auto;
+        width: fit-content;
+    }
+}
```

Comment:
I'm not sure, my concern with the 3x2 layout is that it would be incorrect depending on which way the screen faces. Which, could be fixed with a button to flip the layout. We could also have a setting to switch between the traditional layout and the 3x2 layout. Options are good? Personally I prefer the traditional layout.

Reply:
I agree with options being good, but the main issue that I have with the vertical layout is that it doesn't make good use of the horizontal space of the screen, so you end up with dead space that could instead be used to make the display more visible across the field. The goal here is that, eventually, this would be the single interface for interacting with field statuses. If you look up in the slack channel, you can see the figma designs that I've put together on this.